domains/transfer/Transfer.API/API.IntegrationTests/SwaggerTests.GetSwaggerDocs_v20240103.verified.txt

```diff
@@ -73,8 +73,8 @@
               }
             }
           },
-          "404": {
-            "description": "CVR company not found"
+          "400": {
```

Comment:
This should return a ValidationProblem

Reply:
It already does:

![image](https://github.com/Energinet-DataHub/energy-origin/assets/55732914/0aad87f3-0fb9-4fd8-b658-299d157a389b)